### PR TITLE
First runescape command - ge price lookup

### PR DESCRIPTION
**Pull request by irismessage**

### CODE REVIEW

File: cogs/runescape_commands.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            if value_str.endswith(multi):
+                value_str = value_str.rstrip(multi)
+                value = int(value_str) * multipliers[multi]
+                break
+        else:
+            value = int(value_str)
+
+        return value
+
+    def get_item_value_by_id(self, item_id, return_int=True):
```
Comment (4Kaylum) [inline cogs/runescape_commands.py]: This needs to be an async function

File: cogs/runescape_commands.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        if return_int:
+            if type(value) == str:
+                value = self.rs_notation_to_int(value)
+        else:
+            value = str(value)
+
+        return value
+
+    @utils.command()
+    def ge(self, ctx, item, rs_notation: bool = True):
```
Comment (4Kaylum) [inline cogs/runescape_commands.py]: * This needs to be an async function
* Could you rename it to `grandexchange` and add `ge` as an alias?
* It needs a docstring to show in the bot's help menu

File: cogs/runescape_commands.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        if item_id := self.item_ids.get(item):
```
Comment (4Kaylum) [inline cogs/runescape_commands.py]: This is valid in Python3.9 but I am running on Python3.6 - before the walrus operator. Sorry :<

File: cogs/runescape_commands.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            return await ctx.send(self.get_item_value_by_id(item_id, return_int=not rs_notation))
```
Comment (4Kaylum) [inline cogs/runescape_commands.py]: This needs to be awaited (after changing the function to async).

**Review by 4Kaylum — CHANGES_REQUESTED** [review]

File: cogs/runescape_commands.py (hunk 4 above)
Comment (Silverdimond) [reply cogs/runescape_commands.py:65]: This also uses the walrus operator.

File: cogs/runescape_commands.py (hunk 1 above)
Comment (irismessage) [reply cogs/runescape_commands.py]: got dang

Comment (irismessage) [reply cogs/runescape_commands.py]: c9a4d63a833e4e4815551dc960ce276d0e50fcac

**Review by 4Kaylum — APPROVED** [review]

**Pull request merged**